src/coreclr/jit/codegenwasm.cpp

```diff
@@ -707,6 +711,38 @@ void CodeGen::genCodeForLclVar(GenTreeLclVar* tree)
     }
 }
 
+//------------------------------------------------------------------------
+// genCodeForStoreLclVar: Produce code for a GT_STORE_LCL_VAR node.
+//
+// Arguments:
+//    tree - the GT_STORE_LCL_VAR node
+//
+void CodeGen::genCodeForStoreLclVar(GenTreeLclVar* tree)
+{
+    assert(tree->OperIs(GT_STORE_LCL_VAR));
+
+    if (tree->gtGetOp1()->IsMultiRegNode())
+    {
+        NYI_WASM("multi-reg GT_STORE_LCL_VAR");
+    }
+
+    LclVarDsc* varDsc = compiler->lvaGetDesc(tree);
+
+    if (!varDsc->lvIsRegCandidate())
+    {
+        var_types type = varDsc->GetRegisterType(tree);
+        // TODO-WASM: actually local.get the frame base local here.
+        GetEmitter()->emitIns_S(ins_Store(type), emitTypeSize(tree), tree->GetLclNum(), 0);
```

Comment:
This won't actually work; these stores need to be lowered away since the address is the first operand (at the bottom of the stack). It will be done by RA.